### PR TITLE
Add ublox repo

### PR DESCRIPTION
**Pull request by wep21**
Ublox repo somehow disappeared from repos file, so I added it into repos.

### CODE REVIEW

File: autoware.proj.repos [hunk 1]
```diff
@@ -67,6 +67,10 @@ repositories:
     type: git
     url: https://github.com/tier4/tamagawa_imu_driver.git
     version: ros2
+  vendor/ublox:
+    type: git
+    url: https://github.com/tier4/ublox.git
+    version: tier4/ros2
```
Comment (jilaada) [inline autoware.proj.repos:73]: Should this be `foxy-devel` instead since that is now in sync with upstream?

Comment (wep21) [reply autoware.proj.repos:73]: For now, `foxy-devel` is even with upstream `foxy-devel` and `tier4/ros2` includes a commit which fred made (https://github.com/tier4/ublox/pull/5) ahead of upstream `foxy-devel`

Comment (jilaada) [reply autoware.proj.repos:73]: Ah I see! I forgot about the missing file needed for sensing_launch! Thanks

**Review by jilaada — APPROVED** [review]
LGTM!

**Pull request merged**